benchmarks/torchbench_model.py

```diff
@@ -60,6 +60,7 @@
 
 # Skip the experiment of a model if any of the experiment configs in the list is fully matched
 DENY_LIST = {
+    "cm3leon_generate": [{}],  # no install.py
```

Comment:
Why are we skipping `cm3leon_generate` and `hf_T5_generate` benchmarks? Although they are skipped in the pytorch/benchmark `install.py` script, as far as I remember, they still do run successfully (at least on inductor).

Reply:
Thanks, if we don't care about `install.py`, maybe we just keep the inductor run and skip other configs? I tried model `cm3leon_generate` at  in L4, it tooks more than 1 hour to execute the model without returning results. WDYT?

Reply:
Also I notice that if we set `DEFAULT_TRAIN_BSIZE ` when we install models. If we don't install the model, the test will throw out error: 
```
File "/home/ml-auto-solutions/benchmark/torchbenchmark/util/model.py", line 202, in _determine_batch_size
    raise NotImplementedError(f"Model's {'DEFAULT_TRAIN_BSIZE' if self.test == 'train' else 'DEFAULT_EVAL_BSIZE'} is not implemented.")
NotImplementedError: Model's DEFAULT_TRAIN_BSIZE is not implemented.
```

Reply:
> maybe we just keep the inductor run and skip other configs?

I guess we could do this. According to #5932, both models timeout (after 30 mins).

> If we don't install the model, the test will throw out error

I don't think that's exactly how it works. A model only has an `install.py` file if there are any required dependecies/action needed before running the model (e.g. installing dependencies with `pip`).

That said, some models don't define `DEFAULT_TRAIN_BSIZE`, which probably means that the model wasn't designed for training (guess).

Reply:
I add a new argument `--strict-compatible` (value is default to False which has no impact to current setup) to choose between whether we want to involve those stackdump or timeout models. We don't what to wait for timeout or stackdump since this is break up the benchmark automation system when we do remote execution.

Reply:
Cool. But, maybe we should skip those only when `"xla": "PJRT"`, since inductor runs successfully. What do you think?